From 6e83ea5217f4789f21ce2be706747d7d0aba07f5 Mon Sep 17 00:00:00 2001
From: John Thiltges <jthiltges2@unl.edu>
Date: Mon, 9 Jan 2017 15:45:22 -0600
Subject: [PATCH 3/7] ocserv-fw should send all traffic to the device-specific
 forwarding chain

After adding port-specific rules to FORWARD and creating SEC_FORWARD_CHAIN
with route-specific rules, send any remaining FORWARD traffic to
SEC_FORWARD_CHAIN.
---
 src/ocserv-fw | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/ocserv-fw b/src/ocserv-fw
index 3653364..1c2899a 100755
--- a/src/ocserv-fw
+++ b/src/ocserv-fw
@@ -270,6 +270,10 @@ else
 	iptables -A ${SEC_FORWARD_CHAIN} -i ${DEVICE} -j ACCEPT -m comment --comment "${COMMENT}"
 fi
 
+# send traffic to the route chain
+iptables  -A FORWARD -i ${DEVICE} -j ${SEC_FORWARD_CHAIN} --match comment --comment "${COMMENT}"
+ip6tables -A FORWARD -i ${DEVICE} -j ${SEC_FORWARD_CHAIN} --match comment --comment "${COMMENT}"
+
 execute_next_script
 
 exit 0
-- 
2.1.4

